### PR TITLE
[feature][workflow] Add Pulsar PR Naming Convention

### PR DESCRIPTION
**Pull request by Anonymitaet**
Preview looks good:
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@D-2-Ed @DaveDuggins @momo-jun could u PTAL? Thanks

**Review by Huanli-Meng — APPROVED** [review]
LGTM

File: site2/website/contributing.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ You'll then push to your branch on GitHub. Note: when updating your commit after
 
 Navigate to the [Pulsar GitHub Repo](https://github.com/apache/pulsar) to create a pull request.
 
+> **Note**
+> 
+> Vague, boring, and unclear PR titles decrease team efficiency and productivity. Good titles speed up the review process and increase search efficiency. PR titles should be engaging, easy to understand, and readable. For how to **write self​-explanatory PR titles** and **quality PR title examples**, see [[Guideline] Pulsar PR Naming Convention](https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#).
```
Comment (momo-jun) [inline site2/website/contributing.md:201]: Regarding the word "quality", do you mean to "qualify" (as a verb) or "qualified" (as an adj.)?

**Review by codelipenghui — APPROVED** [review]

**Review by mattisonchao — APPROVED** [review]

**Review by Demogorgon314 — APPROVED** [review]

**Review by RobertIndie — APPROVED** [review]

File: site2/website/contributing.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/website/contributing.md:201]: quality means "excellent" as an adjective
eg. quality work
quality chocolate

**Pull request merged**